.github/workflows/push-to-gar-docker-multiarch.yaml

```diff
@@ -0,0 +1,131 @@
+name: Multi-platform Docker image push to Google Artifact Registry
+
+on:
+  workflow_call:
+    inputs:
+      platforms:
+        default: "['amd64']"
```

Comment:
Would it perhaps be better to also use the same format as we have with the docker/build-push-action?

```
linux/amd64
linux/arm64
...
```